cs/src/Contracts/TunnelSshKeyResponse.cs

```diff
@@ -0,0 +1,19 @@
+// <copyright file="TunnelSshKeyResponse.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// </copyright>
+using System.Text.Json.Serialization;
+
+namespace Microsoft.VsSaaS.TunnelService.Contracts;
+
+/// <summary>
+/// Response for SshKey endpoint.
+/// </summary>
+public class TunnelSshKeyResponse
```

Comment:
Could the SSH gateway public key be a property on `TunnelRelayTunnelEndpoint`, instead of a separate API + contract? It is related to the relay endpoint in that the relay is acting as the SSH gateway. And there are other SSH properties on the relay endpoint object already.

Reply:
... but only if it does not add extra time/cost (such as a keyvault query) to operations that retrieve the endpoint.